### PR TITLE
Russian Peasant Algorithm in C++

### PR DESCRIPTION
**Pull request by mahimagoyalx**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Code for Russian Peasant Algorithm in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #978

### CODE REVIEW

**Comment by mahimagoyalx** [comment]
@HarshCasper @Kajol-Kumari @iamrajiv @siddharth25pandey Kindly review.

File: C-Plus-Plus/cp/RussianPeasant.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Russian Reasant Algorithm
```
Comment (iamrajiv) [inline C-Plus-Plus/cp/RussianPeasant.cpp]: @mahimagoyalx Add more description about Russian Peasant Algorithm.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@mahimagoyalx Move this program to `other` section. Add time complexity.

**Review by iamrajiv — APPROVED** [review]

**Comment by mahimagoyalx** [comment]
@plazzy99 @Abhijit2505 Kindly review.

**Review by vatsalkesarwani12 — APPROVED** [review]

**Comment by vatsalkesarwani12** [comment]
remove merge conflict

**Comment by mahimagoyalx** [comment]
> remove merge conflict

Done

**Comment by mahimagoyalx** [comment]
@Kajol-Kumari Kindly review.

**Review by atarax665 — APPROVED** [review]

**Comment by iamrajiv** [comment]
@all-contributors please add @mahimagoyalx for code

**Pull request merged**